### PR TITLE
[@types/pg] Add Client to PoolConfig

### PR DESCRIPTION
**Pull request by alexgraul**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 

`Client` on `PoolConfig`: https://github.com/brianc/node-postgres/tree/master/packages/pg-pool#create

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/pg/index.d.ts
1. types/pg/pg-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 87a03a00ff1dfb1137b8bb5d43fd87890f69be5e
</p>

**Comment by pspeter3** [comment]
LGTM

File: types/pg/index.d.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export interface PoolConfig extends ClientConfig {
     Promise?: PromiseConstructorLike | undefined;
     allowExitOnIdle?: boolean | undefined;
     maxUses?: number | undefined;
+    Client?: new() => ClientBase | undefined;
```
Comment (gabritto) [inline types/pg/index.d.ts:52]: Perhaps you meant:
```suggestion
    Client?: (new () => ClientBase) | undefined;
```

**Review by gabritto — CHANGES_REQUESTED** [review]

**Comment by alexgraul** [comment]
@gabritto good catch, thankyou! Fixed.

**Review by gabritto — APPROVED** [review]

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/pg/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6c3b9935cbee4d331781e96794d8573ab0559ee6
</p>

**Comment by alexgraul** [comment]
Ready to merge

**Pull request merged**